README.rst

```diff
@@ -20,6 +24,10 @@ To install, check out the source, and use the included setup script, as::
 
    > python setup.py install
 
+You will also need to clone and install the following related Stratis
```

Comment:
It's just that they don't really have to clone...they could use pip, or download a zip file from the arhcives, or something.